src/library/scala/collection/StringOps.scala

```diff
@@ -1,10 +1,12 @@
 package scala
 package collection
 
-import java.lang.StringBuilder
+import java.lang.{StringBuilder => JStringBuilder}
 import java.util.NoSuchElementException
 
 import scala.collection.immutable.{ArraySeq, WrappedString}
+import scala.collection.immutable.{ArraySeq, WrappedString}
+import scala.collection.mutable.StringBuilder
```

Comment:
That is just `scala.StringBuilder`. But I wondered if there were style points in using the `java.lang.StringBuilder` import and then `mutable.StringBuilder` where needed. Just looking for a cooler way besides the trite `J` prefix. The `mutable` is familiar from when distinguishing `immutable`.